lib/Commands.cc

```diff
@@ -266,7 +266,7 @@ SharedBuffer Commands::newConnect(const AuthenticationPtr& authentication, const
     BaseCommand cmd;
     cmd.set_type(BaseCommand::CONNECT);
     CommandConnect* connect = cmd.mutable_connect();
-    connect->set_client_version(PULSAR_VERSION_STR);
+    connect->set_client_version(std::string("Pulsar-C++-v") + PULSAR_VERSION_STR);
```

Comment:
change to "Pulsar-CPP-v"?

Reply:
The documentation prefers to use `C++` over `CPP`. I think `C++` is better for users. https://pulsar.apache.org/docs/next/client-libraries-cpp/

Reply:
Oh. I see that we are already using CPP here: https://github.com/apache/pulsar-client-cpp/blob/05807bdaf3a4341b22efd7c71f7b00c47cc31413/lib/HTTPLookupService.cc#L195

+1 for using CPP